### ISSUE TITLE
SSL: CERTIFICATE_VERIFY_FAILED

### ISSUE THREAD

**Issue by marcianosr** [tejado/pokemongo-api-demo#35]
Hey there! 

I'm trying to run the main.py with: 
`python main.py -a "google" -u "--@gmail.com" -p "password" -l "Washington Square, New York"`

But I get this as error: 

```
Traceback (most recent call last):
  File "main.py", line 247, in <module>
    main()
  File "main.py", line 206, in main
    set_location(args.location)
  File "main.py", line 47, in set_location
    loc = geolocator.geocode(location_name)
  File "c:\Python27\lib\site-packages\geopy\geocoders\googlev3.py", line 217, in geocode
    self._call_geocoder(url, timeout=timeout), exactly_one
  File "c:\Python27\lib\site-packages\geopy\geocoders\base.py", line 171, in _call_geocoder
    raise GeocoderServiceError(message)
geopy.exc.GeocoderServiceError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:590)>
```

I installed the requirements. and I'm using Python 2.7.11. How can I fix this? 

Thanks :)!

**Comment by andrzej-jedrzejewski** [comment]
How did you install the reqs?

`pip2.7 install -r requirements.txt` just works fine for me ;)

**Comment by marcianosr** [comment]
I used `python -m pip install -r requirements.txt`. It says all dependencies are installed. Pip freezes my terminal...

**Comment by andrzej-jedrzejewski** [comment]
what is your OS?

**Comment by marcianosr** [comment]
I'm using Windows 8.1 :-)

**Comment by andrzej-jedrzejewski** [comment]
So maybe try to revert to [https://www.python.org/download/releases/2.6.6/](python 2.6.6)

**Comment by marcianosr** [comment]
Thanks! I fixed it! Still using python 2.7. 

My pip froze because of my avast antivirus program. I just killed it temporary and now it works. Thanks for the help :)
